### PR TITLE
Anti-RightClick: cafe.daum.net 

### PR DESCRIPTION
**Pull request by ghost**
### URLs
https://cafe.daum.net/SoulDresser/FLTB/418592?svc=cafeapi
https://cafe.daum.net/Tlwkftlqkftlldlqkf/8vJr/192317?svc=cafeapi

### Environment
* Operating System: Ubuntu 21.04
* Browser: Firefox Developer Edition 92.0b3

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1311,8 +1311,7 @@ my-expert.ru#$#html { -webkit-touch-callout: default!important; -moz-user-select
...
 page.kakao.com#$#div[id="epubViewerBody"][class^="css-"] { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
-webtoon.kakao.com,page.kakao.com#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
-cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
+cafe.daum.net,webtoon.kakao.com,page.kakao.com#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1314]: What's this rule for? It seems old rules are still needed. Btw, `List-KR` have similar rule.

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1314]: The text of `cafe.naver.com` cannot be copied on Firefox.

Comment (zloyden) [reply AnnoyancesFilter/sections/tweaks.txt:1314]: Ok, I got it. Please additionally revert `cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1314]: I think we don't need to revert before https://github.com/AdguardTeam/AdguardFilters/pull/90884/commits/6cbd0eec2981322088c49196a347050546d1bf28.
https://user-images.githubusercontent.com/37463155/129352277-7cc0b2ae-6db8-421f-a102-cacbef6cb676.mp4
https://user-images.githubusercontent.com/37463155/129352292-fb5b2be6-f602-465e-ba49-26b991c5d399.mp4

Comment (zloyden) [reply AnnoyancesFilter/sections/tweaks.txt:1314]: I meant that I need two rules for text selection.
```
cafe.daum.net#$#body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
cafe.daum.net#$#body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1314]: Okay.

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1315]: Done. https://github.com/AdguardTeam/AdguardFilters/pull/90884/commits/b430fd9411809ac7baa78397a8204b5e2aa7980e

**Review by zloyden — APPROVED** [review]

**Pull request merged**